prime-router/src/main/kotlin/ActionLog.kt

```diff
@@ -0,0 +1,148 @@
+package gov.cdc.prime.router
```

Comment:
There are more of these files that contain data classes.  Let's move this and similar into a good folder and not the root.